doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1262,6 +1262,7 @@ Categorical
 - In meth:`Series.unstack`, specifying a ``fill_value`` not present in the categories now raises a ``TypeError`` rather than ignoring the ``fill_value`` (:issue:`23284`)
 - Bug when resampling :meth:`Dataframe.resample()` and aggregating on categorical data, the categorical dtype was getting lost. (:issue:`23227`)
 - Bug in many methods of the ``.str``-accessor, which always failed on calling the ``CategoricalIndex.str`` constructor (:issue:`23555`, :issue:`23556`)
+- Bug in :mod:`pandas.utils._move` where C variables were declared with external linkage causing import errors if certain other C libraries were imported before Pandas. (:issue:`24113`)
```

Comment:
can you move to the other section, and remove the `` :mod:`pandas.utils._move` ``

Reply:
just remove the link, or don't mention which module the error was in?

Reply:
just remove the module, i think the statement about c libraries is ok (but move it to the 'other' section, at the bottom)